pages/weight.js

```diff
@@ -10,19 +14,82 @@ export async function getServerSideProps() {
 }
 
 export default function WeightPage({ days }) {
+  const canvasEl = useRef(null);
+
+  const filteredDays = days.filter((day) => Boolean(day.weight));
+
+  const ascendingFilteredDays = Array.from(filteredDays).reverse();
```

Comment:
Is it good practice to clone an array, to display it once ascending and descending the other time on the same page?